application/common/sync/Synchronizer.php

```diff
@@ -105,6 +105,12 @@ public function syncAll()
         }
     }
     
+    /**
+     * Synchronize a specific user, requesting their information from the
+     * ID Store and updating it accordingly in the ID Broker.
+     *
+     * @param string $employeeId The EmployeeID of the user to sync.
+     */
     public function syncUser($employeeId)
     {
         $idStoreUser = $this->idStore->getActiveUser($employeeId);
```

Comment:
The ID Sync will internally use field names that match the ID Broker, and each IdStore component will internally define what the ID Store field names are that correspond with the expected field names. For an example, see [InsiteIdStore](https://github.com/silinternational/idp-id-sync/blob/develop/application/common/components/InsiteIdStore.php#L9-L20).